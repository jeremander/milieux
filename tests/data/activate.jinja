{# this is a jinja comment #}

# example template for a custom activation script

echo "Activating environment: {{ENV_NAME}}"
echo "Using Python {{ENV_PYVERSION}}"

# source environment's activation script
source {{ENV_ACTIVATE_PATH}}

# set up custom alias, pointing to environment's bin dir
alias myprog="{{ENV_BIN_DIR}}/my_program"

# regular environment variables do not get substituted during render
export HOME_DIR=$HOME

# make use of a custom variable
echo "CUSTOM_VAR = {{CUSTOM_VAR}}"
