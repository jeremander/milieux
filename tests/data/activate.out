

# example template for a custom activation script

echo "Activating environment: myenv"
echo "Using Python 3.11.2"

# source environment's activation script
source /path/to/myenv/bin/activate

# set up custom alias, pointing to environment's bin dir
alias myprog="/path/to/myenv/bin/my_program"

# regular environment variables do not get substituted during render
export HOME_DIR=$HOME

# make use of a custom variable
echo "CUSTOM_VAR = custom"
